prql-compiler/prqlc/src/cli.rs

```diff
@@ -116,8 +116,12 @@ enum Command {
     SQLCompile {
         #[command(flatten)]
         io_args: IoArgs,
+
+        /// With this option set, the output SQL does not have a signature comment at the bottom
```

Comment:
```suggestion
        /// Exclude the signature comment containing the PRQL version
```

(no strong view, feel free to discard my suggestion)